meta-facebook/at-cb/src/platform/plat_sdr_table.c

```diff
@@ -24,7 +24,3909 @@
 #include "plat_sensor_table.h"
 
 SDR_Full_sensor plat_sdr_table[] = {
```

Comment:
GT isn't planned to use IPMI correct?
Is the SDR table necessary then?

Reply:
Hi @GoldenBug ,
BIC needs to use IPMI to communicate with BMC, but the protocol between BIC and BMC doesn't use IPMB.
BIC needs to record the SDR table to provide the SDR of the sensor to the BMC.